src/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,82 @@
+import * as classnames from "classnames";
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import {Sticky, StickyContainer} from "react-sticky";
+
+import Step from "./Step";
+
+import "./Stepper.less";
+
+const propTypes = {
+  className: PropTypes.string,
+  style: PropTypes.object,
+  steps: PropTypes.arrayOf(PropTypes.shape({
+    title: PropTypes.string.isRequired,
+    description: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+    success: PropTypes.bool,
+    warning: PropTypes.bool,
+    optional: PropTypes.bool,
+  })).isRequired,
+  currentStep: PropTypes.number,
+  seekable: PropTypes.bool,
+  stickySidebar: PropTypes.bool,
+  onStepClick: PropTypes.func,
+};
+
+const defaultProps = {
+  currentStep: 0,
+};
+
+const cssClass = {
+  CONTAINER: "Stepper",
+  SIDEBAR: "Stepper--sidebar",
+  STEPS_DISPLAY: "Stepper--stepsDisplay",
+};
+
+export default class Stepper extends React.PureComponent {
+  static propTypes = propTypes;
+  static defaultProps = defaultProps;
+  static cssClass = cssClass;
+
+  _renderSidebar(style) {
+    const {currentStep, steps, seekable, onStepClick, className} = this.props;
+
+    return (
+      <div style={style}>
+        <ul className={cssClass.STEPS_DISPLAY}>
```

Comment:
`<div>` wrapper here is redundant - the `<div><ul></ul></div>` can be collapsed into a single `<ul>`

Reply:
^still relevant